.github/workflows/release_alpha.yml

```diff
@@ -0,0 +1,147 @@
+name: Build and push image
+
+on:
+  workflow_dispatch:
+    inputs:
+      tag:
+        description: 'Docker tag'
+        required: true
+
+jobs:
+  build:
+    name: Run Gitian build
+    runs-on: self-hosted
+    steps:
+    - name: Checkout
+      uses: actions/checkout@v2
+      with:
+        path: dash
+    
+    - name: Checkout Gitian builder
+      uses: actions/checkout@v2
+      with:
+        repository: devrandom/gitian-builder
+        path: gitian-builder
+
+    - name: Checkout detached sigs
+      uses: actions/checkout@v2
+      with:
+        repository: dashpay/dash-detached-sigs
+        path: dash-detached-sigs
+
+    - name: Checkout gitian sigs
+      uses: actions/checkout@v2
+      with:
+        repository: dashpay/gitian.sigs
+        path: gitian.sigs
+
+    - name: Import GPG key
+      id: import_gpg
+      uses: crazy-max/ghaction-import-gpg@v4
+      with:
+        gpg_private_key: ${{ secrets.GPG_PRIVATE_KEY }}
+        passphrase: ${{ secrets.GPG_PASSPHRASE }}
```

Comment:
do we need a new (ci only) user (`<signer>`) for these?

Reply:
or we need to not sign them, yeah those are the two options

Reply:
I didn't realise not signing was an option, how can that be achieved? I don't think it's necessary to sign alpha builds if they are only running on our own infrastructure anyway, and not signing would allow us to remove some complexity in the CI.

Reply:
@strophy The gitian build script's `-D` option creates a detached file that can be downloaded/signed elsewhere. I guess you could build it that way and just never sign. There may be some other option to ignore signing altogether, but I'm not sure.

Reply:
yeah, you include a name but I don't think it's used for anything in that case